File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_forecast_snapshot.rs


pub fn convert__forecast_snapshot__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::forecast_snapshot::ForecastSnapshot) -> crate::victron_controller::dashboard::forecast_snapshot::ForecastSnapshot {
    crate::victron_controller::dashboard::forecast_snapshot::ForecastSnapshot {
        today_kwh: from.today_kwh.clone(),
        tomorrow_kwh: from.tomorrow_kwh.clone(),
        fetched_at_epoch_ms: from.fetched_at_epoch_ms.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/from_0_1_0_forecast-snapshot.ts
// @ts-nocheck
import {ForecastSnapshot as v0_1_0_ForecastSnapshot} from './v0_1_0/ForecastSnapshot'
import {ForecastSnapshot as dashboard_ForecastSnapshot} from './ForecastSnapshot'

export function convert__forecast_snapshot__from__0_1_0(from: v0_1_0_ForecastSnapshot): dashboard_ForecastSnapshot {
    return new dashboard_ForecastSnapshot (
        from.today_kwh,
        from.tomorrow_kwh,
        from.fetched_at_epoch_ms,
        []
    )
}